eq-author/src/App/settings/ThemesPage.js

```diff
@@ -89,6 +117,51 @@ const ThemesPage = ({ questionnaire }) => {
     }
   };
 
+  const handleEQIdBlur = ({ value }, shortName) => {
+    value = value.trim();
+    if (value !== "") {
```

Comment:
```suggestion
    if (value) {
```

A string is falsy when it is empty (`""`) and truthy otherwise, so you can omit the `!== ""` part of this.